.jenkins/docker_cuda_theano/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install tools
+RUN apt-get update
+RUN apt-get install -y build-essential apt-utils wget git dvipng time
+
+# Setup conda python for Theano
+RUN wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
+RUN bash Miniconda2-latest-Linux-x86_64.sh -p /miniconda -b
+ENV PATH=/miniconda/bin:${PATH}
+RUN conda install numpy scipy sympy mkl nose pydot-ng graphviz cython cmake
+RUN conda update -y conda
+RUN pip install nose-timer parameterized "flake8<3" "sphinx==1.3.5"
```

Comment:
such an old sphinx? travis use 1.5.1. If that old one work, I don't have objection to that. I'm just surprised.